src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -66,8 +66,14 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     case SubmitWorkflow(wdlSource, wdlJson, inputs) =>
       submitWorkflow(wdlSource, wdlJson, inputs, maybeWorkflowId = None) pipeTo sender
     case WorkflowStatus(id) => dataAccess.getWorkflowState(id) pipeTo sender
-      // TODO actually attempt to abort workflow
-    case WorkflowAbort(id) => sender ! Option(WorkflowAborted)
+    case WorkflowAbort(id) =>
+      workflowStore.toMap.get(id) match {
+        case Some(x) =>
+          x ! WorkflowAbort(id)
+          sender ! Some(WorkflowAborting)
+        case None =>
+          sender ! None
```

Comment:
I believe it's because the API uses a tell? Nominate @mcovarr to field this one...